.github/workflows/add_reponame_issue_label.yml

```diff
@@ -0,0 +1,52 @@
+name: Add repo name to issue label
+
+# on:
+#   workflow_dispatch:
+
+on:
+  issues:
+  pull_request:
+  
+env:
+  GH_TOKEN: ${{ secrets.ADD_TO_PROJECT_PAT }}
+
+jobs:
+
+  add-label:
+
+    runs-on: ubuntu-latest
+
+    permissions:
+      issues: write
+
+    steps:
+
+      - name: checkout repo
+        uses: actions/checkout@v4
+        with:
+          ref: 'main'
+          token: ${{ secrets.ADD_TO_PROJECT_PAT }}
```

Comment:
not related to the error you're getting, but I don't think you need to give a custom PAT because the default token is scoped to the current repo

Reply:
ah never mind, it seems a custom PAT is required for labels